services/app-web/src/components/FilterAccordion/FilterAccordion.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import styles from './FilterAccordion.module.scss'
+import { Accordion, Button } from '@trussworks/react-uswds'
+
+export type FilterAccordionPropType = {
+    onClearFilters?: () => void
+    filterTitle: string
+    children: React.ReactElement[]
+}
+
+export const FilterAccordion = ({
+    onClearFilters,
+    filterTitle,
+    children,
+}: FilterAccordionPropType) => {
+    const [toggleClearFilter, setToggleClearFilter] = useState(false)
+
+    const handleClearFilters = () => {
+        setToggleClearFilter(!toggleClearFilter)
+        if (onClearFilters) onClearFilters()
+    }
+
+    const childrenWithToggleProps = React.Children.map(children, (child) => {
+        return React.cloneElement(child as React.ReactElement, {
+            toggleClearFilter,
+        })
```

Comment:
I am controlling the `Clear filters` button in this component instead of the parent component. `toggleClearFilter` state is what updates the children component by passing it in using `React.cloneElement`. This way the entire `FilterAccordion` and `FilterSelect` components are not reliant on the parent and reusable outside of the `HealthPlanPackageTable` component.